lib/API/api_requests.dart

```diff
@@ -0,0 +1,48 @@
+import 'package:http/http.dart' as http;
+import 'dart:convert';
+
+Future<http.Response> getUserLists(String token) {
+  return http.post(
+    Uri.http('10.0.2.2:5000', '/get_lists'),
```

Comment:
Avoid usage of string literals in code files. Move to a configuration file.

Reply:
do you want me to also move the strings that describe the body of the request to a config file? (like in lines 25,26)

Reply:
Yes, that's correct. 
A good rule of thumb would be "If I ever need to change this value, is it possible that other values would need a change?" If the answer is yes then they should all be referencing one single value.
In this case, multiple requests share the key `'token'`